recipes/scikit-image/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [ `uname` == "Darwin" ]
+then
+    export LDFLAGS="-L$PREFIX/lib -Wl,-rpath,$PREFIX/lib"
```

Comment:
Appears this doesn't cut it actually. Try building with Python 3.5 and NumPy 1.10.x (though presumably the problem would occur with NumPy 1.11.x). Here is the [log]( https://api.travis-ci.org/jobs/131975942/log.txt?deansi=true ) for details. Though `-headerpad_max_install_names` seems to still work fine.